content/posts/customizing/dark-theme/index.fr.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Activer le mode sombre"
+date: 2022-06-12T01:30:50+06:00
+author:
+  name: Emruz Hossain
+  image: images/author/emruz.jpg
+menu:
+  sidebar:
+    name: Activer le thème sombre
+    identifier: enable-dark-theme
+    parent: Personnalisation
```

Comment:
Please, use the parent identifier same as the English version.

Reply:
Fixed